qa/common/gen_qa_model_repository

```diff
@@ -512,7 +512,7 @@ chmod -R 777 $FORMATDESTDIR
 python3 $SRCDIR/gen_qa_trt_data_dependent_shape.py --models_dir=$DATADEPENDENTDIR
 chmod -R 777 $DATADEPENDENTDIR
 # Make shared library for custom Hardmax plugin.
-(git clone -b release/10.0 https://github.com/NVIDIA/TensorRT.git && \
+(git clone -b release/10.5 https://github.com/NVIDIA/TensorRT.git && \
```

Comment:
This place better 
```suggestion
docker pull ${TENSORRT_IMAGE}
TENSORRT_VERSION="$(docker inspect ${TENSORRT_IMAGE} --format '{{index .Config.Labels "com.nvidia.tensorrt.version"}}' | cut -d . -f -2)"
(git clone -b release/${TENSORRT_VERSION} https://github.com/NVIDIA/TensorRT.git && \
```

Reply:
Thanks @mc-nv! Ack the suggestions. This is not to be merged and is still under development and is only a work around for generating models on V100 to unblock testing.